lib/index.js

```diff
@@ -175,30 +133,36 @@ function morphAttrs(fromNode, toNode) {
     var attrName;
     var attrValue;
     var foundAttrs = {};
+    var fromValue;
 
-    for (i=attrs.length-1; i>=0; i--) {
+    for (i = attrs.length - 1; i >= 0; i--) {
         attr = attrs[i];
         if (attr.specified !== false) {
```

Comment:
Don't think there's a need to test on specified. If it's an attribute in source it must also be an attribute in target